porcelain/api.go

```diff
@@ -144,18 +144,17 @@ func (a *API) MinerPreviewSetPrice(
 	return MinerPreviewSetPrice(ctx, a, from, miner, price, expiry)
 }
 
-// GetAndMaybeSetDefaultSenderAddress returns a default address from which to
-// send messsages. If none is set it picks the first address in the wallet and
-// sets it as the default in the config.
-func (a *API) GetAndMaybeSetDefaultSenderAddress() (address.Address, error) {
-	return GetAndMaybeSetDefaultSenderAddress(a)
-}
-
 // WalletBalance returns the current balance of the given wallet address.
 func (a *API) WalletBalance(ctx context.Context, address address.Address) (*types.AttoFIL, error) {
 	return WalletBalance(ctx, a, address)
 }
 
+// DefaultWalletAddress returns a default wallet address from the config.
+// If none is set it picks the first address in the wallet and sets it as the default in the config.
+func (a *API) DefaultWalletAddress() (address.Address, error) {
```

Comment:
WalletDefaultAddress would be more idomatic and cause this method to sort with its friends. Would you mind making the update?

Reply:
thanks, agree with you.